code/bash_scripts/load_psql.sh

```
@@ -16,5 +34,5 @@ for shapefile in $(find downloads/ -type f -name '*.shp');
  do
    tablename="$(basename "$shapefile" .shp)"
    echo $tablename
-   shp2pgsql -d -I -s $tabblock_projection $shapefile $schema.$tablename| psql
+   shp2pgsql -d -I -s $tabblock_projection $shapefile $schema.$tablename | psql
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)